diff -u -r linux-5.4.105/net/ipv6/addrconf.c linux-5.4.105-modified/net/ipv6/addrconf.c
--- linux-5.4.105/net/ipv6/addrconf.c	2021-03-11 15:06:51.000000000 +0200
+++ linux-5.4.105-modified/net/ipv6/addrconf.c	2021-03-28 15:37:51.218432176 +0300
@@ -3581,7 +3581,7 @@
 				idev->if_flags |= IF_READY;
 			}
 
-			pr_info("ADDRCONF(NETDEV_CHANGE): %s: link becomes ready\n",
+			pr_debug("ADDRCONF(NETDEV_CHANGE): %s: link becomes ready\n",
 				dev->name);
 
 			run_pending = 1;
