diff -u -r linux-5.4.105/net/core/dev.c linux-5.4.105-modified/net/core/dev.c
--- linux-5.4.105/net/core/dev.c	2021-03-11 15:06:51.000000000 +0200
+++ linux-5.4.105-modified/net/core/dev.c	2021-03-28 15:38:00.279443034 +0300
@@ -1126,9 +1126,6 @@
 		return err;
 	}
 
-	if (oldname[0] && !strchr(oldname, '%'))
-		netdev_info(dev, "renamed from %s\n", oldname);
-
 	old_assign_type = dev->name_assign_type;
 	dev->name_assign_type = NET_NAME_RENAMED;
 
@@ -7668,9 +7665,6 @@
 		}
 	}
 	if (dev->flags != old_flags) {
-		pr_info("device %s %s promiscuous mode\n",
-			dev->name,
-			dev->flags & IFF_PROMISC ? "entered" : "left");
 		if (audit_enabled) {
 			current_uid_gid(&uid, &gid);
 			audit_log(audit_context(), GFP_ATOMIC,
