diff -r -u linux-5.4.72/net/core/dev.c linux-5.4.72-modified/net/core/dev.c
--- linux-5.4.72/net/core/dev.c	2020-10-17 11:11:24.000000000 +0300
+++ linux-5.4.72-modified/net/core/dev.c	2020-10-21 19:46:30.227943309 +0300
@@ -1126,9 +1126,6 @@
 		return err;
 	}
 
-	if (oldname[0] && !strchr(oldname, '%'))
-		netdev_info(dev, "renamed from %s\n", oldname);
-
 	old_assign_type = dev->name_assign_type;
 	dev->name_assign_type = NET_NAME_RENAMED;
 
@@ -7667,9 +7664,6 @@
 		}
 	}
 	if (dev->flags != old_flags) {
-		pr_info("device %s %s promiscuous mode\n",
-			dev->name,
-			dev->flags & IFF_PROMISC ? "entered" : "left");
 		if (audit_enabled) {
 			current_uid_gid(&uid, &gid);
 			audit_log(audit_context(), GFP_ATOMIC,
